alert-common/src/main/java/com/synopsys/integration/alert/common/rest/AlertRestConstants.java

```diff
@@ -3,33 +3,56 @@
  *
  * Copyright (c) 2022 Synopsys, Inc.
  *
- * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All
+ * rights reserved worldwide.
  */
 package com.synopsys.integration.alert.common.rest;
 
 public final class AlertRestConstants {
```

Comment:
This class may have been using the wrong save actions, I don't believe we want to insert spaces between each field